src/data/faq_de.json

```diff
@@ -306,7 +306,7 @@
                         "textblock": [
                             "Gemeinsam mit der EU-Kommission wurde eine Infrastruktur für einen sicheren Informationsaustausch entwickelt, die als sog. \"Interoperability Gateway\" bezeichnet wird. Dieses Gateway ermöglicht, dass die Corona-Warn-App über Landesgrenzen hinaus funktioniert und auch im Ausland potentielle Warnmeldungen empfangen werden können. Detaillierte Informationen zum Gateway bietet die EU-Kommission unter <a href='https://ec.europa.eu/commission/presscorner/detail/de/qanda_20_1905#gateway'>Coronavirus: EU-Datenabgleichsdienst für Kontaktnachverfolgungs- und Warn-Apps – Fragen und Antworten</a>.",
                             "Um weitere Informationen in der App zu sehen, klicken Sie",
-                            "<ol><li>'RISIKO-ERMITTLUNG AKTIV (bzw. GESTOPPT) an'</li><li>Dann auf 'Länderübergreifende Risiko Ermittlung' klicken.</li><li>Als letztes klicken Sie auf 'Teilnehmende Länder'.</li></ol>",
+                            "<ol><li>'RISIKO-ERMITTLUNG AKTIV (bzw. GESTOPPT) an'</li><li>Dann auf 'Länderübergreifende Risiko Ermittlung' klicken.</li><li>Als letztes scorllen Sie nach unten zu der Liste der Länder.</li></ol>",
```

Comment:
In "Als letztes scorllen Sie nach unten zu der Liste der Länder."

- überlegen, ob man auf "Als letztes" in der Liste verzichten kann. Die Nummerierung in der Liste macht es überflüssig, eine Reihenfolge zu beschreiben.
- das Wort "scorllen" korrigieren

Vorschlag:
"Scrollen Sie nach unten zu der Liste der Länder."